### PR TITLE
New Canonical Type for MAU

### PR DESCRIPTION
**Pull request by darshanjain12**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml [hunk 1]
```diff
@@ -632,6 +632,19 @@ MAU_SFSS2X_SFVSC2X_CHWSC_HWSC_FDPM_SSPC_STC_VOADM2X:
   - SSPC
   - STC
   - VOADM2X
+  
+MAU_SFVSC_SPSS_STM_SHM_SFC:
+  description: "MAU with VFD ,spray pump ,filter , supply air temperature and huidity monitoring"
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/MAU.yaml:637]: nit: fix comma position.

**Review by tasodorff — DISMISSED** [review]
fix nit

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml (hunk 1 above)
Comment (darshanjain12) [reply ontology/yaml/resources/HVAC/entity_types/MAU.yaml]: Apologies. Done the  required changes

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml [hunk 2]
```diff
@@ -632,6 +632,19 @@ MAU_SFSS2X_SFVSC2X_CHWSC_HWSC_FDPM_SSPC_STC_VOADM2X:
   - SSPC
   - STC
   - VOADM2X
+  
+MAU_SFVSC_SPSS_STM_SHM_SFC:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/MAU.yaml:636]: Please add `SFSS`

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml [hunk 3]
```diff
@@ -632,6 +632,19 @@ MAU_SFSS2X_SFVSC2X_CHWSC_HWSC_FDPM_SSPC_STC_VOADM2X:
...
+  is_canonical: true
+  implements:
+  - MAU
+  - SFVSC
+  - SPSS
+  - STM
+  - RTM
+  - SFC
+  opt_uses:
+  - filter_differential_pressure_status
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/MAU.yaml:647]: use type `FDPSM` instead of adding single field

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml (hunk 3 above)
Comment (darshanjain12) [reply ontology/yaml/resources/HVAC/entity_types/MAU.yaml]: Done

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml (hunk 2 above)
Comment (darshanjain12) [reply ontology/yaml/resources/HVAC/entity_types/MAU.yaml]: Done

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml [hunk 4]
```diff
@@ -632,6 +632,20 @@ MAU_SFSS2X_SFVSC2X_CHWSC_HWSC_FDPM_SSPC_STC_VOADM2X:
   - SSPC
   - STC
   - VOADM2X
+  
+MAU_SFSS_SFVSC_SPSS_STM_SHM_SFC_FDPSM:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/MAU.yaml:636]: please double-check this type name, it does not match the listed types in the implements section

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml (hunk 4 above)
Comment (darshanjain12) [reply ontology/yaml/resources/HVAC/entity_types/MAU.yaml:636]: Done

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**